FordLabs/retroquest
# Batch Delete Archived Boards

## Overview
Added the ability to batch delete archived action item boards (group of thoughts).

### Demo
<img width="1492" alt="Screen Shot 2022-11-10 at 11 22 14 AM" src="https://user-images.githubusercontent.com/17144844/201150559-b630f2f3-051b-4464-b680-f5605d466c3b.png">

<img width="1509" alt="Screen Shot 2022-11-10 at 11 22 07 AM" src="https://user-images.githubusercontent.com/17144844/201150568-ad4d2779-cdf1-4183-8d5a-bf952ca8e227.png">

## Testing Instructions
1) Archive a number of boards so the archived thoughts page has items on it.
2) Select one or two of the boards via the checkbox on the left, and ensure a "Delete Selected" button shows up.
3) Click that button and ensure a modal pops up. Clicking "Cancel" should close modal and do nothing. Clicking "Yes, Delete" should delete the selected items.
4) Click the select all checkbox on the gray bar. Ensure all remaining action items are selected.
5) Deselect one and ensure the select all checkbox gets unselected.
6) Reselect the last board and ensure the select all button gets re checked.
